readme.md

```diff
@@ -9,7 +9,44 @@ Brushing teeth too soon after eating or drinking can soften the tooth enamel, es
 
 Breathing deeply in moments of stress, or anytime during the day, brings many benefits such as better circulation, decreased anxiety and reduced 
 blood pressure1.
+#random
+here are some random health facts:

```

Comment:
Consider adding a table of contents at the beginning of the readme.md file for easier navigation. [medium]